doc/man3/OBJ_nid2obj.pod

```diff
@@ -130,6 +130,8 @@ These functions cannot return B<const> because an B<ASN1_OBJECT> can
 represent both an internal, constant, OID and a dynamically-created one.
 The latter cannot be constant because it needs to be freed after use.
 
+All of these functions are thread safe.
```

Comment:
Do we actually need to say this? Isn't that assumed for all functions? I view it more as a bug that they weren't before.

Reply:
We're variable about what are and what aren't thread safe.  I figured documenting that these are was useful.
Happy to remove it.

Reply:
Where functions don't conform with our "normal" thread safety pattern (as documented in openssl-threads(7)) then we should call that out in the individual man pages. With the changes in this PR, this brings the code into line with what we do everywhere else so I see no reason to mention it.

Reply:
I would move this into HISTORY section. `These functions were made thread safe in OpenSSL 3.0.`

Reply:
I've added the line to HISTORY.